apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -1,14 +1,12 @@
-import {fireEvent, render, screen} from '@testing-library/react';
+import {fireEvent, render, screen, waitFor} from '@testing-library/react';
+import '@testing-library/jest-dom';
 import React from 'react';
 import {Provider} from 'react-redux';
+import sinon from 'sinon'; // eslint-disable-line no-restricted-imports
```

Comment:
Why is sinon needed?

Reply:
Said another way -- using jest is preferred. Can jest be used or is there a reason sinon must be used here?

Reply:
Oh I can switch to jest!